engines/director/detection_tables.h

```diff
@@ -3660,6 +3661,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1("fukuda", "", "Mac/FUKUDA",  "0b55676d539ba5e29560e7c4da4fbad6", 483187, 404),
 	WINGAME1t("fukuda", "", "WIN/MAIN/FUKUDA.EXE",  "71eec68c19354b67badc7c82f159374b", 739951, 404),
 
+	MACGAME1_l("fukuokagoround", "", "Fukuoka Go-Round",  "0c7bbb4b24823e5ab871cb4c1d6f3710", 484167, Common::JA_JPN, 404),
+	WINGAME1t_l("fukuokagoround", "", "FGR.EXE",  "dc2da038ba67c2bd2f5bb31b97261db9", 5293611, Common::JA_JPN, 404),
```

Comment:
A three-letter EXE is quite generic, maybe include an additional file?

Reply:
Good call, done.